src/utilities/browser-utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Allows us to use the documentMode for feature detection on globalThis.Document
+ */
+declare global {
+  interface Document {
+      documentMode?: any;
+  }
+}
+/**
+* Returns true if the detected browser is Internet Explorer.
+* @returns boolean
+*/
+const _isIE = (): boolean => {
+  return /*@cc_on!@*/ false || !!document.documentMode;

```

Comment:
While you won't be able to get branch coverage for `/*@cc_on!@*/`, you could mock document.documentMode to test this function. If time permits, please write some unit tests for these methods.

Reply:
Added tests.